package.json

```diff
@@ -132,7 +132,7 @@
     "babel-core": "^6.6.4",
     "babel-plugin-external-helpers": "^6.5.0",
     "babel-polyfill": "^6.6.1",
-    "babel-preset-react-native": "^1.7.0",
+    "babel-preset-react-native": "frantic/babel-preset-react-native",
```

Comment:
Remove this, please. We will have to update this seperately (i.e. after publishing to npm)